src/commands/bitpos.json

```diff
@@ -58,31 +58,25 @@
                         "type": "integer"
                     },
                     {
-                        "name": "end_index",
-                        "type": "block",
```

Comment:
can you help me figure out what happened here?

Reply:
actually this is a "bugfix": before this commit the "unit" argument used to be dependent on the existence of the "end" argument, which is not true according to the code (and common sense)
used to be
```BITPOS key bit [ start [ end [ BYTE | BIT]]]```
and not it's in the correct form of
```BITPOS key bit [ start [ end ] [ BYTE | BIT]]```

Reply:
added to top comment

Reply:
i don't think that's right.
looking at the command's code (and the comment above the command proc), it's only valid if `c->argc == 6`.

Reply:
yes, my bad, i'll revert